### DISCUSSION TITLE
Translation fields

### DISCUSSION THREAD

**Discussion by TomasSestak** [directus/directus#4382] [Archive]
Hi i'm looking for the solution with the relational translation fields but the documentation just says yet to come, is it already possible and just the documentation is in progress or is is still WIP feature?

btw. thank you for your awesome work, Directus 9 looks pretty cool so far

**Comment by rijkvanzanten** [comment]
Heya! Translations are very much already a part of Directus. Where in the docs did you find that it's coming soon? That must've been an oversight / left over from earlier times 🙂

**Reply by TomasSestak** [reply]
hello, thank you for quick response, here's the link: https://docs.directus.io/guides/field-types/translated-fields/

**Reply by rijkvanzanten** [reply]
Ahh yeah.. the documentation is to come, but the feature already exists 🙂 

You can go into data model > collection > create field and click "Translations" to get started 👍🏻 

![CleanShot 2021-03-04 at 10 08 29@2x](https://user-images.githubusercontent.com/9141017/109984273-997c4780-7cd1-11eb-8723-2030f4dabb9c.png)

**Reply by benhaynes** [reply]
I was going to go write these Docs now, but since our Field Setup flow is about to change, I'll wait until that change is in place.

**Reply by TomasSestak** [reply]
Ok, when it's about to change i'll throw something in: in dropdown, you can't control language of the options
